src/main/java/games/strategy/triplea/TripleAPlayer.java

```diff
@@ -682,7 +682,7 @@ public void confirmOwnCasualties(final GUID battleId, final String message) {
     ui.getBattlePanel().confirmCasualties(battleId, message);
   }
 
-  public final boolean isDamageFromBombingDoneToUnitsInsteadOfTerritories(final GameData data) {
+  private static boolean isDamageFromBombingDoneToUnitsInsteadOfTerritories(final GameData data) {
```

Comment:
This method is only called within this class, so I reduced it's accessibility to `private`.